content/cpp/concepts/operator-overloading/operator-overloading.md

```diff
@@ -0,0 +1,111 @@
+---
+Title: 'Operator Overloading'
+Description: 'Operator overloading redefines operators for user-defined classes.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Operators'
+  - 'Binary'
+  - 'Logical'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+**Operator overloading** redefines operators for user-defined classes. When an overloaded operator is called, the compiler determines which definition to use based on the arguments provided. The following operators can be overloaded:
+
+| ->      | ->\* | new    | new [] | delete | delete []     |
+| ------- | ---- | ------ | ------ | ------ | ------------- |
+| +       | -    | \*     | /      | %      | ^             |
+| +=      | -=   | \*=    | /=     | %=     | ^=            |
+| &#124;= | &    | &#124; | ~      | !      | ,             |
+| =       | &=   | <<=    | >>=    | []     | ()            |
+| <       | >    | <=     | >=     | ++     | --            |
+| <<      | >>   | ==     | !=     | &&     | &#124; &#124; |
+
+The operators below cannot be overloaded:
+
+| ::  | .\* | .   | ?   |
```

Comment:
Why is this a table rather than a bulleted list?

Reply:
@SSwiniarski Should I change both tables to a bulleted list?

Reply:
@KTom101 I think it would be less confusing.